tests/Microsoft.Bot.Builder.TestBot.Json/Samples/ToDoLuisBot/ToDoLuisBot.lg

```diff
@@ -15,7 +15,7 @@
 - * @{x}
 
 # ShowItems
-- IF: {listName == "shopping"}
+- IF: @{listName == "shopping"}
```

Comment:
I hope we can commit a real sample showing very flexible multi-locale support, 

let's say
a.lg  // import [common.lg]
a.zh_cn.lg // import [common.lg]

common.lg // import [b.lg]
b.lg
b.zh_cn.lg.

I mean at least we should show things like this to make sure our solution is covered well